main.go

```diff
@@ -47,7 +48,11 @@ func main() {
 		config.Cmd = strings.Split(os.Getenv("CMD"), " ")
 	}
 	create := docker.CreateContainerOptions{Name: containerName, Config: &config}
-	hostConfig := docker.HostConfig{PublishAllPorts: true}
+
+	portBindings := map[docker.Port][]docker.PortBinding{
+        docker.Port(publishPort + "/tcp"): []docker.PortBinding{docker.PortBinding{HostPort: publishPort }}}
```

Comment:
I think that maybe we could just have two separate env variables, one for which port to expose and one for which port to *expose on*? Then the port to expose on should always include the protocol